doc/api/prev_api_changes/api_changes_3.1.0.rst

```diff
@@ -750,9 +750,6 @@ The following signature related behaviours are deprecated:
 - Passing ``shade=None`` to `~.axes3d.Axes3D.plot_surface` is deprecated. This
   was an unintended implementation detail with the same semantics as
   ``shade=False``. Please use the latter code instead.
-- `matplotlib.ticker.MaxNLocator` and its *set_params* method will issue
```

Comment:
This should stay as it refers to the code at https://github.com/matplotlib/matplotlib/pull/13992/files#diff-efde3b8d2ec4015b0fc910f1080cc492L2020 which has not ben reverted.